.github/workflows/run_system_tests.yml

```diff
@@ -36,11 +36,17 @@ jobs:
           path: |
             packages/service/.venv
             packages/service/.tox
-          key: run-system-tests-${{ runner.os }}-${{ matrix.configuration }}-${{ hashFiles('packages/service/poetry.lock') }}
-      - name: Install dependencies
+          key: run-system-tests-${{ runner.os }}-${{ matrix.configuration }}-${{ hashFiles('packages/service/poetry.lock') }}-${{ hashFiles('packages/generator/poetry.lock') }}
```

Comment:
Don't do this. Cache the service and generator venvs separately.

Reply:
Sure, I have separated the venvs for service and generator.